enterprise-suite/Makefile

```diff
@@ -1,27 +1,20 @@
+CHART ?= enterprise-suite
+RELEASE_NAME = es
+
+include ../common.mk
+
+
 test: lint
 
 lint: lint-helm lint-json lint-promql
 
-lint-helm:
-	helm lint .
-
 lint-json:
-	find . -name \*.json | xargs -tn 1 jq . >/dev/null
+	find $(CHART_DIR) -name \*.json | xargs -tn 1 jq . >/dev/null
 
 lint-promql:
-	scripts/validate-promql.sh
+	$(CHART_DIR)/scripts/validate-promql.sh
```

Comment:
Not clear to me why CHART_DIR is needed here? make should already be running from the chart directory.

Reply:
This is a concession to making things work for enterprise-suite-latest.